src/core/generator/from_proto.rs

```diff
@@ -308,8 +308,6 @@ impl Context {
                 }
             }
 
-            ty.tag = Some(Tag { id: msg_type.id() });
```

Comment:
We should drop `Namespace::id` method and other unnecessary logic from `Namespace` that isn't being used any more.

Reply:
it is used ig